src/settings/LocationLocations/LocationManager.js

```diff
@@ -276,27 +370,31 @@ class LocationManager extends React.Component {
         return reject(errors);
       });
     });
-  }
+  };
 
   onChangeInstitution = (e) => {
     this.setState({ institutionId: e.target.value, campusId: null });
-  }
+  };
 
   onChangeCampus = (e) => {
     this.setState({ campusId: e.target.value, libraryId: null });
-  }
+  };
 
   onChangeLibrary = (e) => {
     this.setState({ libraryId: e.target.value });
-  }
-
-  filterRow(row) {
-    return (row.libraryId === this.state.libraryId);
-  }
+  };
 
   renderFilter() {
-    const { resources, intl: { formatMessage } } = this.props;
-    const { institutionId, campusId, libraryId } = this.state;
+    const {
+      resources,
+      location,
+      intl: { formatMessage },
+    } = this.props;
+    const {
+      institutionId,
+      campusId,
+      libraryId,
+    } = this.state;
```

Comment:
It looks like a blank line should be added after this line for code consistency

Reply:
Agree. Done.